include/multipass/id_map.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2021 Canonical, Ltd.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; version 3.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef MULTIPASS_ID_MAP_H
+#define MULTIPASS_ID_MAP_H
+
+#include <utility>
+#include <vector>
+
+namespace multipass
+{
+typedef typename std::vector<std::pair<int, int>> id_map;
```

Comment:
Done. I changed the name of the data structure, as you suggest. However, I didn't change the names `uid_map` and `gid_map` because I think those names make the interfaces easier to understand. WDYT?